src/appleseed-max-impl/utilities.cpp

```diff
@@ -39,6 +39,7 @@
 #include "renderer/api/color.h"
 #include "renderer/api/source.h"
 #include "renderer/api/texture.h"
+#include "renderer/modeling/texture/tileptr.h"
```

Comment:
I'll make a PR on appleseed's side to include `renderer/modeling/texture/tileptr.h` into `renderer/api/texture.h`.

Reply:
True, should be part of the api, then adding '#include "renderer/modeling/texture/tileptr.h"'can be omitted.